src/types/onyx/PrivatePersonalDetails.ts

```diff
@@ -5,6 +5,9 @@ type Address = {
     state: string;
     zip: string;
     country: string;
+    zipPostCode?: string;
+    addressLine1?: string;
+    addressLine2?: string;
```

Comment:
@situchan Also making this props as optional and `GetPhysicalCardUtils` removal those are the only changes

Reply:
Done